index.html

```diff
@@ -192,6 +192,38 @@ <h3>Skills</h3>
         </div>
         
     </section>
+
+    <!-- Contact Section -->
+    <section class="contact-section">
+        <h2>I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.</h2>
```

Comment:
- [ ] Great design :100: There is just something I would like to suggest to make the overall look and feel better :wink: I noticed that the nav bar is fixed and as a result of that when you scroll and you reach this part of the contact section, the texts on the nav and the contact section overlap. Making it difficult to make out the words in each of them. :heavy_check_mark: 
- 
![image](https://user-images.githubusercontent.com/102699958/213014814-82eceeca-8a9b-4229-8e0c-aca9d97b39bc.png)
- This isn't very great in terms of user interaction so I would suggest that you either make the nav bar a darker shade of white or not make it fixed at all so it doesn't show when the user gets to that part of the page. :+1: